.github/workflows/validate-no-offensive-lang.yml

```diff
@@ -0,0 +1,15 @@
+name: validate-no-offensive-lang
+
+on:
+  pull_request:
+    branches: [ master ]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
```

Comment:
Just checking if there is a fedora/centos environment available as that's what our own stuff runs in I believe.

Reply:
Good question, but unfortunately ubuntu is the only linux which is available on github hosted runners:  https://docs.github.com/en/actions/using-github-hosted-runners/about-github-hosted-runners#supported-runners-and-hardware-resources .